src/Bridge/OpenAI/GPT/ResponseConverter.php

```diff
@@ -33,7 +34,17 @@ public function convert(HttpResponse $response, array $options = []): LlmRespons
             return $this->convertStream($response);
         }
 
-        $data = $response->toArray();
+        try {
+            $data = $response->toArray();
+        } catch (\Throwable $e) {
```

Comment:
in this case maybe catching only `ClientExceptionInterface`?

Reply:
done